keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -109,6 +109,9 @@ SIG Architecture for cross-cutting KEPs).
     - [Reserve](#reserve)
     - [Unreserve](#unreserve)
```

Comment:
I believe you are working on some improvements in scheduler in this cycle. Can you please summarize in the KEP?

Reply:
The event handling changes already went into 1.28. I'll check how to document that here.

We were discussing moving the blocking API calls out of the main scheduling cycle, but I am with @Huang-Wei: let's not do that and instead just document the effect. Will add that here.